feature/mypage/src/main/java/com/teamsolply/solply/mypage/MypageContract.kt

```
@@ -1,13 +1,22 @@
 
 data class MypageState(
-    val town: String = "연희동"
+    val town: String = "연희동",
+    val nickname: String = "",
 ) : UiState
 
-sealed interface MypageIntent : UiIntent
+sealed interface MypageIntent : UiIntent {
 
-sealed interface MypageSideEffect : SideEffect
+}
```

detekt (1.23.8)

[warning] The class or object MypageIntent is empty.

(detekt.empty-blocks.EmptyClassBlock)